frontend/src/Cadastro.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import './Cadastro.css';
+
+const Cadastro: React.FC = () => {
+  const [numeroOS, setNumeroOS] = useState<number>(1);
+  const [dataEntrada, setDataEntrada] = useState<string>("");
+
+  useEffect(() => {
+    // Busca o último número salvo e soma 1
+    const ultimoNumero = localStorage.getItem("ultimoNumeroOS");
```

Comment:
O número da ordem de serviço será gerado pelo Backend, ou seja, só será obtido quando a ordem de serviço for salva. 

Remover essa implementação que gera e armazena o número da ordem de serviço